Jenkinsfile

```diff
@@ -3,34 +3,55 @@ properties([
     durabilityHint('PERFORMANCE_OPTIMIZED'),
     buildDiscarder(logRotator(numToKeepStr: '5')),
 ])
-parallel kind: {
-    node('docker') {
-        timeout(90) {
-            checkout scm
-            withEnv(["WSTMP=${pwd tmp: true}"]) {
-                try {
-                    sh 'bash kind.sh'
-                    dir (WSTMP) {
-                        junit 'surefire-reports/*.xml'
-                    }
-                } finally {
-                    dir (WSTMP) {
-                        if (fileExists('kindlogs/docker-info.txt')) {
-                            archiveArtifacts 'kindlogs/'
+
+def splits
+stage('Determine splits') {
+    node('linux') {
```

Comment:
May want to use a container agent.

Reply:
Yeah I wasn't sure of the right syntax in the context of `ci.jenkins.io`.

Reply:
You can use e.g. `maven-11`.